src/server/handlers/validator.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: MIT
+//! Expose the subnet actor validator set
+
+use crate::config::ReloadableConfig;
+use crate::lotus::client::LotusJsonRPCClient;
+use crate::lotus::LotusClient;
+use crate::server::JsonRPCRequestHandler;
+use anyhow::anyhow;
+use async_trait::async_trait;
+use cid::Cid;
+use ipc_sdk::subnet_id::SubnetID;
+use ipc_subnet_actor::Validator;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use std::sync::Arc;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct QueryValidatorSetParams {
+    pub subnet: String,
+    pub tip_set: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct QueryValidatorSetResponse {
+    /// The address of the created subnet
+    pub validator_set: Vec<Validator>,
```

Comment:
We should introduce the new `ValidatorSet`struuct introduced in this PR: https://github.com/consensus-shipyard/ipc-actors/pull/61. We need all of it for reconfiguration.

Reply:
@adlrocha I have updated the types, please take a look.